Frontend/src/Modals/FormModal.js

```diff
@@ -67,16 +69,20 @@ function FormModal(props) {
     const isFormValid = formValidation();
 
     if (isFormValid) {
-      axios
-        .post(`http://localhost:8000/memo_places/places/`, place, {
-          headers: {
-            'Content-Type': 'application/json',
-          },
-        })
-        .then(() => {
+      if (props.type === 'update') {
+        axios
+          .put(`http://localhost:8000/memo_places/places/place&id=${updatePlaceData.place.id}`, {
```

Comment:
A tutaj to co to za przyjemniaczek raczej powinno być
`http://localhost:8000/memo_places/places/place/id=${updatePlaceData.place.id}` 
jeśli rozumiem o co ci chodzi. Aktualizacja może jeszcze nie działać perfekcyjnie bo tego nie testowałem

Reply:
to tak wstępnie wpisałem bo wiem, że jeszcze nad tym pracujesz, jak tylko skończysz to poprawie tak jak powinno być